client/containers/Booking/Booking.jsx

```diff
@@ -61,44 +58,42 @@ export class Booking extends Component {
                 {
                     this.props.appointment.isSubmitted ?
                         <div className="requestDiv">
-                            <p className="requestText">Your Request has been submitted successfully for Approval</p>
+                            <p className="requestText">You will receive an email once your request is reviewed</p>
                         </div>
                         : ''
                 }
                 {
-                    this.props.appointment.error ?
-                        <div className="requestDiv">
-                            <p className="requestText">Booking is already under Approval process</p>
-                        </div>
-                        : ''
-                }
-                {
-                    this.state.user && this.state.isLogin && !this.props.appointment.isSubmitted && !this.props.appointment.error && !this.state.DarshanApproved ?
+                    (!this.props.appointment.isSubmitted && !this.props.appointment.appointmentData && !this.props.appointment.appointmentData.Appointment) ?
                         <div>
-                            <div className="progressBarDiv">
+                            {/* <div className="progressBarDiv">
                                 <Progress percent={50} />
-                            </div>
+                            </div> */}
                             <div className="bookingformDiv">
+                                <p className="bookingForm">Booking Form</p>
                                 <BookingForm
                                     user={this.state.user ? this.state.user : ''}
                                     createAppointment={this.props.createAppointment} />
                             </div>
                         </div>
                         :
-                        ''
-                }
-                {
-                    this.state.DarshanApproved ?
-                        <iframe
-                            src={`https://nrs15.youcanbook.me/?service=${this.state.DarshanApproved}&skipHeaderFooter=true&noframe=true`}
-                            id="ycbmiframeniranjanaswami"
-                            className="bookingStyle"
-                            frameBorder="0"
-                            allowtransparency="true">
-                        </iframe>
-                        : ''
+                        (
+                            (this.props.appointment.appointmentData && this.props.appointment.appointmentData.Appointment && !this.props.appointment.appointmentData.Appointment.approved &&
+                                !this.props.isSubmitted) ?
+                                < div className="requestDiv">
+                                    <p className="requestText">Your request is under reviewed, you will receive an email once your request is reviewed</p>
```

Comment:
Just paste this text:
Your request is under review.
You will receive an email once your request is reviewed.

Reply:
made the changes, please have a look